File: backend-dotnet/DTO/ManufacturerDTO.cs
namespace backend_dotnet.DTO
{
    public class ManufacturerDTO
    {

        public int ManufacturerId { get; set; }

        public string ManufacturerName { get; set; }


        public ManufacturerDTO()
        {
        }


        public ManufacturerDTO(int manufacturerId, string manufacturerName)
        {
            ManufacturerId = manufacturerId;
            ManufacturerName = manufacturerName;
        }
    }
}

File: backend/java/Vehicle_Configurator/src/main/java/com/example/demo/dto/ManufacturerDTO.java
package com.example.demo.dto;


public class ManufacturerDTO {

    private int manufacturerId;
    private String manufacturerName;

    public ManufacturerDTO() {
    }

    public ManufacturerDTO(int manufacturerId, String manufacturerName) {
        this.manufacturerId = manufacturerId;
        this.manufacturerName = manufacturerName;
    }

    public int getManufacturerId() {
        return manufacturerId;
    }

    public void setManufacturerId(int manufacturerId) {
        this.manufacturerId = manufacturerId;
    }

    public String getManufacturerName() {
        return manufacturerName;
    }

    public void setManufacturerName(String manufacturerName) {
        this.manufacturerName = manufacturerName;
    }
}